Simplified/Adobe DRM/NCXParser.swift

```diff
@@ -0,0 +1,85 @@
+/*
+ 
+ This file is a part of R2Streamer framework;
```

Comment:
crazy idea, probably stupid.... would it work to do `@testable import r2-streamer-swift` wherever you need this class? Since NCXParser is declared as internal. I don't know if swift allows it outside of a testing context.

Reply:
gotcha, I just created https://jira.nypl.org/browse/SIMPLY-2840 for this cleanup. We have a convention of adding a  `TODO: SIMPLY-2840` note in a comment wherever that particular ticket requires intervention, so here would be a good place. If you can make sure to add that note in other places where cleanup is needed after R2 completes multiple DRM support, that would be fantastic.

Reply:
it actually is a good idea, I tested the app with `@testable import R2Streamer` and didn't notice any side effect of that. The only file with @testable statement is `Simplified/Reader2/Internal/LibraryService.swift`. removed that files